graviti/manager/commit.py

```diff
@@ -308,6 +313,7 @@ def from_response(cls: Type[_NC], dataset: "Dataset", contents: Dict[str, Any])
 
         """
         obj = cls(dataset, contents["name"], contents["commit_id"])
+        contents["committed_at"] = from_iso_to_datetime(contents["committed_at"])
```

Comment:
the input argument is contents, here the contents is updated, means the input argument is changed, shall copy content first?